### ISSUE TITLE
What are the training parameters?

### ISSUE THREAD

**Issue by light82** [matterport/Mask_RCNN#1368]
I wonder if I can get the training parameters to train the mask rcnn model,such as training batchsize,learning rate and so on.Thanks a lot.

**Comment by freezurbern** [comment]
The default training parameters are here:
https://github.com/matterport/Mask_RCNN/blob/master/mrcnn/config.py

**Comment by light82** [comment]
Thanks very much.@freezurbern
I have another question.If  I train the model from 0-initial-state,not the pre-trained model,and train the batchnormalization layer, what batchsize should I use?

**Comment by freezurbern** [comment]
I can't comment on batchsize or training from zero. I only use starting from COCO and a batchsize of 1.

**Comment by light82** [comment]
Thanks any way
